webapp/masterclasses.py

```diff
@@ -48,23 +81,45 @@ def get_upcoming_sessions():
         .order_by(nullslast(UpcomingSession.date.desc()))
         .all()
     )
-    for i in range(len(session)):
-        if session[i].date:
-            session[i].date = format_date(session[i].date)
+    for s in session:
```

Comment:
```suggestion
    for session in session_list:
```
I think the naming here is off, session is actually a session list. If we properly name it, we can also properly name the individual session.

Reply:
How about we name it `upcoming_sessions`/`previous_sessions` instead of `session_list` ? That way we're more descriptive of what's inside the variable and also the plurality  alludes to the data type being a list?

Reply:
sounds good!